contributed_definitions/nyaml/NXcs_io_obj.yaml

```diff
@@ -5,7 +5,7 @@ symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
 type: group
-NXcs_io_obj(NXobject):
+NXcs_io_obj(NXcircuit):
```

Comment:
This is a bit more tricky since it also has the `technology` and `max_physical_capacity` which are not necessarily sensible elements for an `NXcircuit`. But you could also say:
```yaml
NXcircuit(NXobject):
  [...]
  technology(NX_CHAR):
    doc: |
      Qualifier for the type of storage medium used (if the circuit is a storage medium). 
    enumeration: [solid_state_disk, hard_disk, tape]
  max_physical_capacity(NX_NUMBER):
    doc: |
      Total amount of data which the medium can hold (if the circuit is a storage medium). 
    unit: NX_ANY
```
and then get rid of `NXcs_io_obj` as well.

Reply:
Agree, followed your suggestion and refactored this set of base classes, only NXcs_computer remains using NXcircuit.